Makefile

```diff
@@ -752,6 +752,12 @@ openstack_deploy_cleanup: namespace netconfig_deploy_cleanup ## cleans up the se
 	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f - || true
 	${CLEANUP_DIR_CMD} ${OPERATOR_BASE_DIR}/openstack-operator ${DEPLOY_DIR}
 
+.PHONY: openstack_fake_update_run
+openstack_fake_update_run:
+	$(eval $(call vars,$@,openstack))
+	bash scripts/openstack-fake-update.sh
```

Comment:
I don't think the target should have the word 'fake' in it. It's off-putting, and if you extend this target to do a 'real' update then it's not a good target name. This does a real update - whether ut;s fake or not depends on the content used, not this process.

Reply:
fixed